app/templates/client/app/app(coffee).coffee

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-angular.module '<%= scriptAppName %>', [<%= angularModules %>]
+angular.module '<%= scriptAppName %>', [<%- angularModules %>]
```

Comment:
I don't think I've seen this one before. What does the `-` do differently from the `=`?